packages/pyxis-react/src/stories/components/CheckboxCard/Overview.tsx

```diff
@@ -169,13 +194,15 @@ const classDescription = (
 
 const Overview: FC = () => (
   <>
-    <OverviewTemplate category="Component" description={overviewDescription} isMain title="Checkbox Card Group">
+    <OverviewTemplate category="Component" description={overviewDescription} isMain title="Checkbox Card">
       <Canvas>
-        <Story id="components-control-checkboxcardgroup-all-stories--default" />
+        <Story id="components-control-checkboxcard-all-stories--default" />
```

Comment:
Questo continua a essere
```suggestion
        <Story id="components-control-checkboxcardgroup-all-stories--default" />
```
altrimenti la pagina si rompe non trovando il file